compiler/control/OMROptions.cpp

```diff
@@ -4551,6 +4593,7 @@ OMR::Options::setAddressEnumerationBits(char *option, void *base, TR::OptionTabl
       }
    else
       {
+      TR_VerboseLog::vlogAcquire();
```

Comment:
The verbose log acquire and release here could be localized to the conditions that would actually require writing to the log file. Alternatively, you could change the `writeLine` to `writeLineLocked`

Reply:
writelinelocked is better